SEED-platform/seed
# refactor: evaluate derived columns in smaller batch size

#### Any background context you want to provide?
SEED's inventory list batches evaluation of derived columns. uWSGI didn't like how we were doing it b/c of too large header (see issue)

#### What's this PR do?
- reduces batch size to **100** to avoid too large of headers. uWSGI's current limit of 4096 bytes means we should be safe until our *View id's are 30+ digits.
  - Explanation: we put *View IDs in the query params as a list (eg `derived_columns/evaluate/<derived column id>/?inventory_ids=123,456,789`), uWSGI's 4096 (available buffer) / 100 (number of IDs passed) = 409.6 -> we could pass up to 100 IDs with 409 digits (bytes). For more context, on dev1 we're at ~10**7 for IDs (~8 digits).

#### How should this be manually tested?
- create a derived column, add it to the inventory list column profile, view it in the inventory list and verify it worked.

#### What are the relevant tickets?
#2794 

#### Screenshots (if appropriate)
